### PR TITLE
Added differences between Briefcase and Aggregate

### LINKED ISSUE #17: Differences between Aggregate and Briefcase
The practical difference between Aggregate and Briefcase is often clear — for example, many features are available in one but not the other — but I've struggled with the conceptual difference. Where does Aggregate end and Briefcase begin? For example, when do you export data from Aggregate vs. export from Briefcase?

I discussed with Yaw and Hélène, who described the following differences and suggested I create an issue in the docs repo:
* Briefcase is good for offline use, for when there's no need for a server. Briefcase is often enough for smaller projects. However, note that Aggregate can also be used offline and locally.
* Briefcase can be used to back up all forms and submissions on Aggregate. In particular, Briefcase's command line interface makes this easier.
* Briefcase is needed to transfer data between Aggregate installs.
* Briefcase is required to decrypt any encrypted submissions.

### PR DESCRIPTION
**Pull request by ankita240796**
Closes #17

### CODE REVIEW

File: briefcase-vs-aggregate.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
+**********************************
+Briefcase vs Aggregate
+**********************************
```
Comment (adammichaelwood) [inline briefcase-vs-aggregate.rst]: Please change this to the underline style:

```
Title
=======
```

The over-under style is for larger docs that *might* be broken up in the future. 
When the over-under style is used, the first subsection headline with the `===` underline.

File: briefcase-vs-aggregate.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+.. _briefcase-points:
+
+Briefcase
+-------------
+
+- Briefcase provides a mechanism to pull blank and finalized forms from ODK Collect or ODK Aggregate or any custom ODK directory into a local briefcase storage location.
```
Comment (adammichaelwood) [inline briefcase-vs-aggregate.rst]: capitalize `Briefcase`
(here, and throughout)

File: index.rst [hunk 3]
```diff
@@ -52,6 +52,7 @@ For a complete list of our projects, check out `Open Data Kit on Github <https:/
    briefcase-guide
    briefcase-install
    briefcase-forms
+   briefcase-vs-aggregate.rst
```
Comment (adammichaelwood) [inline index.rst]: remove file extension in TOC directive

**Comment by ankita240796** [comment]
I made the requested changes.

**Pull request merged**